test/models/location/location-data-mapper-impl.test.ts

```diff
@@ -121,9 +121,13 @@ describe('TEST: Location Data Mapper', () => {
     // @ts-ignore
     it('causes the location to get deleted', async () => {
       // GIVEN: A location with a valid ID to read from
-      const uid = 'test uid';
+      const uid = 1;
+      const testLocation = new Location({
+        locationName: 'Test Location Name',
+        uid: uid,

```

Comment:
This can be shorthanded (TSLint)
```suggestion
        uid,
```